extensions/mssql/src/objectManagement/ui/serverPropertiesDialog.ts

```diff
@@ -80,6 +80,7 @@ export class ServerPropertiesDialog extends ObjectManagementDialogBase<Server, S
 					'margin': '-10px 0px 0px -10px'
 				}
 			}).component();
+		this.disposables.push(serverPropertiesTabbedPannel);
```

Comment:
Something to think about, but it seems like when we're adding components to a container like this the caller shouldn't need to be responsible for disposing it. That should happen automatically when the container itself is disposed (which should happen automatically when the dialog is closed in this case)

Reply:
In this case, formContainer is 'extension-only' component, and is not tracked by ModelViewPanel (i.e. Dialog in this case). ModelViewPanel only calls the delegate to register content: 'registerContent' but if views added inside it are not disposed by extension/callers, they will be leaked even if ModelViewPanel disposes the View they are included in. I've moved this disposable to the respective 'registerContent' method.

Reply:
But thanks for capturing this, I will now track all 'registerContent' calls as they contain untracked components by Model View Impl.